.github/workflows/pytest.yml

```diff
@@ -64,6 +64,34 @@ jobs:
         with:
           terraform_version: "1.5.0"
           terraform_wrapper: false
+      - name: Install Swift with swiftly (macOS)
```

Comment:
Have you checked whether there's a setup-swift action on the GH action marketplace?